cli/hse_cli.c

```diff
@@ -945,13 +932,15 @@ cli_hse_kvdb_compact(struct cli_cmd *self, struct cli *cli)
         .optionv =
             {
                 OPTION_HELP,
+                { "-f, --full", "Full compaction" },
```

Comment:
Perhaps start the help comment with a verb similar to the others.

Reply:
I don't think we support anything except full compaction now.   @gsramdasi ?

Reply:
We still have the default variant which compacts the kvdb until samp is under lwm.